### PR TITLE
Create folder recursively

### PR DESCRIPTION
**Pull request by gnieto**
- Apply rustfmt
- Create the output folder recursively

### CODE REVIEW

File: src/model/owned/table_type/entry.rs [hunk 1]
```diff
@@ -164,8 +163,7 @@ impl ComplexEntry {
             out.write_u8(e.get_type()).unwrap();
 
             // Value
-            out.write_u32::<LittleEndian>(e.get_value() as u32)
-                .unwrap();
+            out.write_u32::<LittleEndian>(e.get_value() as u32).unwrap();
```
Comment (Razican) [inline src/model/owned/table_type/entry.rs:166]: Just checking these unwraps, could we avoid them?

**Comment by Razican** [comment]
Could we add some path/error information here too? https://github.com/gnieto/abxml-rs/blob/b2c3af0e7fd4f6d8d39a3b4e31440b1c86a125dd/src/apk.rs#L43

File: src/model/owned/table_type/entry.rs (hunk 1 above)
Comment (gnieto) [reply src/model/owned/table_type/entry.rs:166]: Mmm I don't remember why I did this, but I will review it anyway, sure. Thanks!

Comment (gnieto) [reply src/model/owned/table_type/entry.rs:166]: Fixed on: https://github.com/gnieto/abxml-rs/pull/53

Originally, as I was writting to a Vec, I thought it was overkill returning a Result, but I think it's much better now. Thanks again for the feedback.

**Pull request merged**